pandas_gbq/gbq.py

```diff
@@ -300,14 +315,41 @@ def load_user_account_credentials(self):
 
         return _try_credentials(self.project_id, credentials)
 
+    def get_default_credentials_path(self):
+        """
+        Gets the default path to the BigQuery credentials
+
+        .. versionadded 0.3.0
+
+        Returns
+        -------
+        Path to the BigQuery credentials
+        """
+
+        import os
+
+        if os.name == 'nt':
+            config_path = os.environ['APPDATA']
+        else:
+            config_path = os.path.join(os.path.expanduser('~'), '.config')
+
+        config_path = os.path.join(config_path, 'pandas_gbq')
```

Comment:
You know this better than I do, but is there a case for having the default path be the default gcloud path, rather than anything specific to pandas?